docs/releases.rst

```diff
@@ -4,7 +4,16 @@
     Releases
 ======================
 
-fmf-1.4
+
+fmf-1.5.0
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Now it is possible to store metadata in hidden files and
+directories as well. Use a simple :ref:`config` file to specify
+names which should be included in the search.
```

Comment:
Store metadata part feels a bit technical for this change. Was thinking something along the lines of fmf tree can now be built with hidden node paths to reflect that `/path/.hidden/something` is the main thing that is now supported.

Reply:
Wording updated in ecd8e0f.